packages/cli/src/commands/teams/switch.ts

```diff
@@ -0,0 +1,127 @@
+// Packages
+import chalk from 'chalk';
+
+// Utilities
+import Client from '../../util/client';
+import listInput from '../../util/input/list';
+import getUser from '../../util/get-user';
+import getTeams from '../../util/get-teams';
+import { Team, GlobalConfig } from '../../types';
+import { writeToConfigFile } from '../../util/config/files';
+
+const updateCurrentTeam = (config: GlobalConfig, team?: Team) => {
+  if (team) {
+    config.currentTeam = team.id;
+  } else {
+    delete config.currentTeam;
+  }
+
+  writeToConfigFile(config);
+};
+
+export default async function change(client: Client, desiredSlug?: string) {
+  const { config, output } = client;
+  const personalScopeSelected = !config.currentTeam;
+
+  output.spinner('Fetching teams information');
+  const [user, teams] = await Promise.all([getUser(client), getTeams(client)]);
+
+  const currentTeam = personalScopeSelected
+    ? undefined
+    : teams.find(team => team.id === config.currentTeam);
+
+  if (!personalScopeSelected && !currentTeam) {
+    output.error(`You are not a part of the current team anymore`);
+    return 1;
+  }
+
+  if (!desiredSlug) {
+    const teamChoices = teams
+      .slice(0)
+      .sort((a, b) => {
+        return a.name.toLowerCase().localeCompare(b.name.toLowerCase());
+      })
+      .map(({ id, slug, name }) => {
+        let title = `${slug} (${name})`;
+        const selected = id === currentTeam?.id;
```

Comment:
If we show the slug first in the list, wouldn't it make sense to also sort by it instead of by name?

Reply:
I think we should show name first. I'll swap.